### PR TITLE
CI setup

### PR DESCRIPTION
**Pull request by follan**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -61,3 +61,7 @@ typings/
 .next
 
 .idea/
+
+build
+
+apps
```
Comment (f1cognite) [inline .gitignore]: Add new line

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  yarn
+  yarn build
+  cp -a ./build/. ../apps/$app
+  cd ..
+done
+
+jsArray=[\"$( IFS=, ; echo "${applications[*]}" )\"]
+jsArray="${jsArray//,/\",\"}"
+jsArray="${jsArray//.\/}"
+sed -i "s/@AppsToken/$jsArray/g" apps/index.html
```
Comment (f1cognite) [inline build.sh]: Add new line

File: .gitignore (hunk 1 above)
Comment (follan) [reply .gitignore]: Done

File: build.sh (hunk 2 above)
Comment (follan) [reply build.sh]: Done

**Pull request merged**